wagtail/admin/search.py

```diff
@@ -30,11 +29,23 @@ def __init__(
         else:
             self.attr_string = ""
 
+    def __eq__(self, other):
+        return (self.order, self.label) == (other.order, other.label)
+
+    def __ne__(self, other):
+        return (self.order, self.label) != (other.order, other.label)
```

Comment:
Apparently `__ne__` defaults to the inverse of `__eq__`, so I think we don't need to implement it

https://docs.python.org/3/reference/datamodel.html#object.__ne__